.changeset/config.json

```diff
@@ -5,12 +5,7 @@
   "linked": [],
   "access": "public",
   "baseBranch": "main",
-  "ignore": [
```

Comment:
q: do u plan to release `@xstate/graph` as a stable release **soon**? we probably should add `@xstate/graph` back here and do some cleanup in https://github.com/statelyai/xstate/tree/7c6e2eaa4e5021738efec345b51f59a8e2de662e/.changeset-xstate-test-beta and only then execute the final release

Reply:
Yes, I plan to release it soon

Reply:
#4911